scripts/install-mender.sh

```diff
@@ -15,6 +15,9 @@
 set -e
 
 CHANNEL="stable"
+if [[ "$*" == *"experimental"* ]]; then
+    CHANNEL="experimental"
+fi
```

Comment:
This defeats `parse_args` function. Instead, isn't it possible to change the order of the main calls so that we do `init` before `check_dist_and_version`?

This would break for example if in the future we have a `something-experimental` package or `experimental` string appearing in some other way in the CLI args.